Formula/xcenv.rb

```diff
@@ -18,9 +18,8 @@ def install
   end
 
   def caveats; <<-EOS.undent
-    Xcenv stores data under ~/.xcenv by default. If you absolutely need to
-    store everything under Homebrew's prefix, include this in your profile:
-      export XCENV_ROOT=#{var}/xcenv
+    To enable shims and autocompletion, run this and follow the instructions:
+      xcenv init
```

Comment:
Please remove these `caveats` as this is not Homebrew-specific information. Thanks!